Gemfile

```diff
@@ -26,7 +26,7 @@ group :development, :test do
   gem 'rake'
   gem 'rdoc'
   gem 'rspec', '~> 3.0'
-  gem 'rubocop', '~> 0.59', require: false
+  gem 'rubocop', '~> 0.60', require: false
```

Comment:
This is OK for now, but really should just be a specific version, because each new release (eg. 0.61) will produce new errors/warnings.

Reply:
Not sure whether it is desirable to lock the gem, rather than tackle the new errors / warnings as they come.
If you still feel I should lock this to a version, I'll do that

Reply:
I got this in https://github.com/ruby-grape/grape-swagger/pull/719.

We lock rubocop because it's otherwise a moving target. Yesterday `~> 0.59` resolved to 0.59, but today it resolved to 0.60, producing new errors. So the same code built green yesterday but fails to build today, which is not desirable. 

New errors and warnings should totally be tackled, but not in the way of feature/fix work :)